foundations/foundation-vertx/src/main/java/org/apache/servicecomb/foundation/vertx/http/FileUploadPart.java

```diff
@@ -36,7 +36,10 @@ public FileUploadPart(FileUpload fileUpload) {
 
   @Override
   public InputStream getInputStream() throws IOException {
-    return Files.newInputStream(new File(fileUpload.uploadedFileName()).toPath());
+    final InputStream inputStream = Files.newInputStream(new File(fileUpload.uploadedFileName()).toPath());
+    final InputStreamWrapper inputStreamWrapper = new InputStreamWrapper(inputStream);
+    FileUploadStreamRecorder.getInstance().recordOpenStream(inputStreamWrapper);

```

Comment:
How about moving this line into the constructor of `InputStreamWrapper`?
It's not about functionality, but since the `clearRecorder` method is invoked in the `InputStreamWrapper.close()` method, maybe it's better to move the `recordOpenStream` method invocation into the `InputStreamWrapper`, too, so that the `FileUploadStreamRecord` can be encapsulated better.

Reply:
fixed